### ISSUE TITLE
Wrong tarball for release 0.7.15 ?

### ISSUE THREAD

**Issue by guillomovitch** [sukria/Backup-Manager#126]
Hello.

The [release page](https://github.com/sukria/Backup-Manager/releases) content is not consistent with project history: release 0.7.14 is listed first, and tagged by github as 'latest release', whereas release 0.7.15 is listed second.

Moreover, the content of release 0.7.15 tarball seems wrong, for instance lacking the fixes introduced by [PR#99](https://github.com/sukria/Backup-Manager/pull/99) and advertised as merged in Changelog for release 0.7.14.

I guess the release tarball for 0.7.15 has been made from a wrong branch (master instead of devel).

**Comment by margamanterola** [comment]
Indeed, the last released version is wrong. It doesn't contain what's listed in the release notes, those changes were made in the devel branch, but not released.
